lib/activeadmin-orderable.rb

```diff
@@ -19,7 +19,7 @@ def orderable
     end
 
     module TableMethods
-      HANDLE = '&#x2195;'.html_safe
+      HANDLE = '&#9776;'.html_safe
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.